### PR TITLE
feat(inoreader): init

### PR DESCRIPTION
**Pull request by Thomas-Philippot**
## 🎉 Theme for Inoreader 🎉

Inoreader is a RSS online reader

catwalk

## 💬 Additional Comments 💬

Don't know if this is the best way to theme but it works fine.

*still needs some tweaks here and there*

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [submission guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/userstyle-creation.md).
- [x] I have made a new directory underneath `/styles/<name-of-website>` containing the contents of the [`/template`](https://github.com/catppuccin/userstyles/blob/main/template/) directory.
  - [x] I have ensured that the new directory is in **lower-kebab-case**.
  - [x] I have followed the template and kept the preprocessor as [LESS](https://lesscss.org/#overview).
- [x] I have made sure to update the
      [`userstyles.yml`](https://github.com/catppuccin/userstyles/blob/main/scripts/userstyles.yml)
      file with information about the new userstyle.
- [x] I have included the following files:
  - `catppuccin.user.css` - all the CSS for the userstyle, based on the
    template.
  - `assets/mocha.webp`, `assets/macchiato.webp`, `assets/frappe.webp` and
    `assets/latte.webp` - individual screenshots of the themed website, in all 4
    Catppuccin flavors.
  - `catwalk.webp` - image of all individual screenshots stitched together,
    generated via [Catwalk](https://github.com/catppuccin/toolbox/tree/main/catwalk#readme).

### CODE REVIEW

File: scripts/userstyles.yml [hunk 1]
```diff
@@ -695,3 +698,9 @@ userstyles:
       app-link: "https://nextjs.org"
       current-maintainers: [*Dandraghas]
 # yaml-language-server: $schema=https://raw.githubusercontent.com/catppuccin/userstyles/main/scripts/userstyles.schema.json
+  inoreader:
+    name: inoreader
+    category: productivity
+    readme:
+      app-link: "https://inoreader.com"
+      current-maintainers: [ *thomas-philippot ]
```
Comment (isabelroses) [inline scripts/userstyles.yml:706]: can you place this section of code in alphabetical order like the rest of the themes.

**Review by isabelroses — COMMENTED** [review]
Some pages:

File: styles/inoreader/catppuccin.user.css [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+  *  like Latte, you can use a `when` guard clause. */
+  & when (@lookup = latte) {
+  /* your code for latte */
+}
+
+  /*  You can also negate the statement above by using
+  *  a `when not` guard clause. */
+  & when not (@lookup = latte) {
+  /* your code for frappe, macchiato and mocha */
+}
```
Comment (uncenter) [inline styles/inoreader/catppuccin.user.css:213]:
```suggestion
```

File: styles/inoreader/catppuccin.user.css [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+     you can conditionally enable light/dark flavors by targeting whatever attribute that website uses to indicate the theme.
+     Un-comment this block if that is the case.
+      For an actual example of this, see https://github.com/catppuccin/userstyles/blob/39288b593278e5efa48007d5fc001292cf910672/styles/youtube/catppuccin.user.css#L21-L30.
+  :root[data-theme="dark"] {
+    #catppuccin(@darkFlavor, @accentColor);
+  }
+  :root[data-theme="light"] {
+    #catppuccin(@lightFlavor, @accentColor);
+  }
+  */
```
Comment (uncenter) [inline styles/inoreader/catppuccin.user.css:237]:
```suggestion
  */
```

File: styles/inoreader/catppuccin.user.css [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+@description Soothing pastel theme for inoreader
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire*", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+/* If you need any help with the information above, please see: https://github.com/openstyles/stylus/wiki/Writing-UserCSS#var */
```
Comment (uncenter) [inline styles/inoreader/catppuccin.user.css:17]:
```suggestion
```

File: styles/inoreader/catppuccin.user.css [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+
+/* For help with URL rules, please see https://github.com/openstyles/stylus/wiki/Writing-styles#valid--moz-document-rules. */
```
Comment (uncenter) [inline styles/inoreader/catppuccin.user.css:19]:
```suggestion
```

File: styles/inoreader/catppuccin.user.css [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+  }
+
+  #sb_reading_part, #wraper.tree_pane_docked #sb_tree_part {
+    box-shadow: none;
+  }
+
+  .whitebutton {
+    background-color: @red;
+    color: @crust;
+    border-style: none;
```
Comment (uncenter) [inline styles/inoreader/catppuccin.user.css:172]: Do we have to hide the border or? Same for the other `border-style: none;`.

File: styles/inoreader/catppuccin.user.css [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+  }
+
+  .add_content_simulated_button_menu, .profile_menu_plan_badge {
+    background-color: @surface0;
+    border-color: @surface0;
+    color: @text;
+  }
+
+  .catalog_follow_featured_collection {
+    border-color: @text;
```
Comment (uncenter) [inline styles/inoreader/catppuccin.user.css:198]: `@text` is usually too dark of a border-color imo, how does it look?

File: styles/inoreader/catppuccin.user.css [hunk 8]
```diff
@@ -0,0 +1,276 @@
...
+    color: @mantle;
+  }
+
+  #preferences_header .icon16, #preferences_dialog_close_button span.icon16 {
+    color: @mantle;
+  }
+
+  .article_tile.article_unreaded.article_expanded {
+    background-color: @base;
+    box-shadow: 0 1px 8px rgba(0, 0, 0, 0.08), 0 1px 3px rgba(0, 0, 0, 0.10);
```
Comment (uncenter) [inline styles/inoreader/catppuccin.user.css:240]: Unthemed box shadow?

**Review by uncenter — DISMISSED** [review]

File: styles/inoreader/catppuccin.user.css (hunk 6 above)
Comment (Thomas-Philippot) [reply styles/inoreader/catppuccin.user.css:170]: image

Found a way to mimic the original button

image

File: styles/inoreader/catppuccin.user.css (hunk 7 above)
Comment (Thomas-Philippot) [reply styles/inoreader/catppuccin.user.css:196]: It is pretty dark indeed

**Comment by uncenter** [comment]
We appreciate your patience! There are some changes we've introduced since you began this PR that need to be backported to this branch:

- Only the Catwalk-generated/composite images are stored in the repository now. That means you can delete the individual flavor screenshots (`assets/latte.webp`, `assets/frappe.webp`, `assets/mocha.webp`, and `assets/macchiato.webp`).
- As there is now only one image, `catwalk.webp`, in the `assets/` folder (from the previous change), we have also flattened the structure - this means the `assets/catwalk.webp` image should now be simply `preview.webp` (in the root of the subdirectory, next to the `catppuccin.user.css` file).

**Comment by uncenter** [comment]
I've updated your PR to address those changes as mentioned in my previous comment. This should be good to merge (I think)!

**Review by isabelroses — APPROVED** [review]

**Comment by isabelroses** [comment]
> I've updated your PR to address those changes as mentioned in my previous comment. This should be good to merge (I think)!

I would agree.

**Pull request merged**